### DISCUSSION TITLE
How can I use queryContent in server/api directory?

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#19238] [Questions] [answered]
So I was hoping that I could get my content through the API server in nuxt, and hoping to server it as an API.. that way I can use it on other sites. How to do it?

**Comment by Baroshem** [comment, marked as answer]
Hey, check out this discussion https://github.com/nuxt/nuxt/discussions/18636 and specifically this comment https://github.com/nuxt/nuxt/discussions/18636#discussioncomment-5067637
